addons/knowledge/static/src/js/widgets/knowledge_dialogs.js

```diff
@@ -157,15 +155,15 @@ const MoveArticleToDialog = Dialog.extend({
              * @param {Function} escapeMarkup
              */
             formatResult: (result, container, query, escapeMarkup) => {
-                const { text } = result;
+                const { text, subject } = result;
                 const markup = [];
                 window.Select2.util.markMatch(text, query.term, markup, escapeMarkup);
                 if (result.id === 'private') {
                     const src = escapeMarkup(this.getLoggedUserPicture());
                     markup.unshift(`<img src="${src}" class="rounded-circle mr-1"/>`);
-                } else if (typeof result.icon !== 'undefined') {
-                    const icon = result.icon || '📄';
-                    markup.unshift(`<span class="mr-1">${escapeMarkup(icon)}</span>`);
+                }
+                if (subject) {
+                    markup.push(`<span class="test-ellipsis text-muted small">  -  ${escapeMarkup(subject[1])}</span>`);
```

Comment:
Ok, don't do that but properly check for root_id / root_name beforehand. Also note that if root_article_id is False it does not return a tuple (id, name_get) but simply False. Code should be made a bit more robust.